gradle.properties

```diff
@@ -46,6 +46,7 @@ protobufVersion=3.15.3
 protoGoogleCommonProtosVersion=2.0.1
 javaPoetVersion=1.13.0
 shadowPluginVersion=4.0.4
+pitestPluginVersion=1.6.0
```

Comment:
This is a test dependency, consider moving it under the `Test dependencies` section  , before/after `spotbugsPluginVersion`.